internal/pkg/cli/run_local.go

```diff
@@ -116,29 +117,31 @@ type runLocalVars struct {
 type runLocalOpts struct {
 	runLocalVars
 
-	sel                 deploySelector
-	ecsClient           ecsClient
-	ecsExecutor         ecsCommandExecutor
-	ssm                 secretGetter
-	secretsManager      secretGetter
-	sessProvider        sessionProvider
-	sess                *session.Session
-	envManagerSess      *session.Session
-	targetEnv           *config.Environment
-	targetApp           *config.Application
-	store               store
-	ws                  wsWlDirReader
-	cmd                 execRunner
-	dockerEngine        dockerEngineRunner
-	repository          repositoryService
-	prog                progress
-	orchestrator        containerOrchestrator
-	hostFinder          hostFinder
-	envChecker          versionCompatibilityChecker
-	debounceTime        time.Duration
-	newRecursiveWatcher func() (recursiveWatcher, error)
-
+	sel            deploySelector
+	ecsClient      ecsClient
+	ecsExecutor    ecsCommandExecutor
+	ssm            secretGetter
+	secretsManager secretGetter
+	sessProvider   sessionProvider
+	sess           *session.Session
+	envManagerSess *session.Session
+	targetEnv      *config.Environment
+	targetApp      *config.Application
+	store          store
+	ws             wsWlDirReader
+	cmd            execRunner
+	dockerEngine   dockerEngineRunner
+	repository     repositoryService
+	prog           progress
+	orchestrator   containerOrchestrator
+	hostFinder     hostFinder
+	envChecker     versionCompatibilityChecker
+	debounceTime   time.Duration
+	dockerExcludes []string
+
+	newRecursiveWatcher  func() (recursiveWatcher, error)
 	buildContainerImages func(mft manifest.DynamicWorkload) (map[string]string, error)
+	readDockerignoreFile func(fs afero.Fs, contextDir string) ([]string, error)
```

Comment:
Why does this need to be a field for `runLocalOpts`, instead of us just calling `dockerfile.ReadDockerignor(...)` at L257?

Reply:
I was thinking it needed to be mocked, didn't realize that I moved the only other call into a different mocked function!